azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageVersionDaoImpl.java

```diff
@@ -49,13 +61,28 @@
 
   private final ImageTypeDao imageTypeDao;
 
-  private static String INSERT_IMAGE_VERSION_QUERY =
-      "insert into image_versions ( path, description, "
-          + "version, type_id, state, release_tag, created_by, modified_by) "
-          + "values (?, ?, ?, ?, ?, ?, ?, ?)";
-  private static String SELECT_IMAGE_VERSION_BASE_QUERY = "select iv.id, iv.path, iv.description, "
-      + "iv.version, it.name, iv.state, iv.release_tag, iv.created_on, iv.created_by, iv.modified_on, "
-      + "iv.modified_by from image_versions iv, image_types it where it.id = iv.type_id";
+  private static final String INSERT_IMAGE_VERSION_QUERY =
+      "insert into image_versions ( path, description, version, type_id, state, release_tag, "
+          + "created_by, created_on, modified_by, modified_on) "
+          + "values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
+  private static final String SELECT_IMAGE_VERSION_BASE_QUERY =
+      "select iv.id, iv.path, iv.description, "
+          + "iv.version, cast(replace(iv.version, '.', '') as unsigned integer) as int_version, "
+          + "it.name, iv.state, iv.release_tag, iv.created_on, iv.created_by, iv.modified_on, "
+          + "iv.modified_by from image_versions iv, image_types it where it.id = iv.type_id";
+  private static final String SELECT_LATEST_ACTIVE_IMAGE_VERSION_QUERY = "select outer_tbl.id, "
```

Comment:
As these are very huge queries, can you please add comment to explain about it? It will be good for long time.